system/plugins/gosymbolextractor/fake_extractor.py

```diff
@@ -0,0 +1,194 @@
+from system.plugins.gosymbolextractor.extractor import GoSymbolExtractor
+from system.helpers.utils import getScriptDir
+
+
+class FakeGoSymbolExtractor(GoSymbolExtractor):
+	def __init__(self):
+		GoSymbolExtractor.__init__(self)
+
+	def setData(self, data):
+		self.project = "github.com/golang/example"
+		self.commit = "729b530c489a73532843e664ae9c6db5c686d314"
+		self.ipprefix = "github.com/golang/example"
+		self.directory = "%s/example" % getScriptDir(__file__)
+		self.noGodeps = []
+
+		self.input_validated = True
+		self.godeps_on = False
+
+	def getData(self):
+		return GoSymbolExtractor.getData(self)
+
+	def execute(self):
+
```

Comment:
Instead of hard-coding all the jsons into the function, you can put both artefacts into json files and read them instead.